rmgpy/rmg/mainTest.py

```diff
@@ -43,136 +43,143 @@
+
+        self.assertTrue(os.path.exists(os.path.join(seedDir, name, 'dictionary.txt')))  # dictionary file made
+        self.assertTrue(os.path.exists(os.path.join(seedDir, name, 'reactions.py')))  # reactions file made
+
+    def testRMGSeedLibraryCreation(self):
+        """Test that seed mechanisms are created in the correct database locations."""
+        self.assertTrue(os.path.exists(self.seedThermo))
+        self.assertTrue(os.path.exists(self.seedKinetics))
+
+    def testRMGSeedEdgeLibraryCreation(self):
+        """Test that edge seed mechanisms are created in the correct database locations."""
+        self.assertTrue(os.path.exists(self.seedThermo))
+        self.assertTrue(os.path.exists(self.seedKinetics))
+
+    def testRMGSeedWorks(self):
+        """Test that the created seed libraries work.
+
+        Note: Since this test modifies the class level RMG instance,
+        it can cause other tests to fail if run out of order."""
+        # Load the seed libraries into the database
+        self.rmg.database.load(
+            path=self.databaseDirectory,
+            thermoLibraries=['testSeed'],
+            reactionLibraries=['testSeed'],
+            seedMechanisms=['testSeed'],
+            kineticsFamilies='none',
+            kineticsDepositories=[],
+            depository=False
+        )
         
         self.rmg.reactionModel = CoreEdgeReactionModel()
-        self.rmg.reactionModel.addReactionLibraryToEdge('testSeed') #try adding seed as library
-        self.assertTrue(len(self.rmg.reactionModel.edge.species)>0)
-        self.assertTrue(len(self.rmg.reactionModel.edge.reactions)>0)
+        self.rmg.reactionModel.addReactionLibraryToEdge('testSeed')  # try adding seed as library
+        self.assertTrue(len(self.rmg.reactionModel.edge.species) > 0)
+        self.assertTrue(len(self.rmg.reactionModel.edge.reactions) > 0)
         
         self.rmg.reactionModel = CoreEdgeReactionModel()
-        self.rmg.reactionModel.addSeedMechanismToCore('testSeed') #try adding seed as seed mech
-        self.assertTrue(len(self.rmg.reactionModel.core.species)>0)
-        self.assertTrue(len(self.rmg.reactionModel.core.reactions)>0)
-        
-        
+        self.rmg.reactionModel.addSeedMechanismToCore('testSeed')  # try adding seed as seed mech
+        self.assertTrue(len(self.rmg.reactionModel.core.species) > 0)
+        self.assertTrue(len(self.rmg.reactionModel.core.reactions) > 0)
+
+    def testRMGSeedEdgeWorks(self):
+        """Test that the created edge seed libraries work.
+
+        Note: Since this test modifies the class level RMG instance,
+        it can cause other tests to fail if run out of order."""
+        # Load the seed libraries into the database
+        self.rmg.database.load(
+            path=self.databaseDirectory,
+            thermoLibraries=['testSeed_edge'],
+            reactionLibraries=['testSeed_edge'],
+            seedMechanisms=['testSeed_edge'],
+            kineticsFamilies='none',
```

Comment:
Sorry, #1155 starts loading reactions as their original sources (template reactions and original library reactions).  If the family or library use to create a given reaction isn't loaded it can't find the original template/library and errors.  I mean I can change it later in that pull request of course, but I think it would be cleaner to do it here.